src/Knp/Snappy/AbstractGenerator.php

```diff
@@ -599,6 +598,25 @@ protected function executeCommand(string $command): array
         ];
     }
 
+    /**
+     * Executes the given command via shell and returns the complete output as
+     * a string.
+     *
+     * @param string[] $command
+     *
+     * @return array [status, stdout, stderr]
+     */
+    protected function executeCommand(array $command): array
+    {
+        if (\method_exists(Process::class, 'fromShellCommandline')) {
+            $process = Process::fromShellCommandline($command, null, $this->env);
+        } else {
+            $process = new Process($command, null, $this->env);
+        }
+
+        return $this->runProcess($process);
+    }
```

Comment:
This method is not called anymore and could be removed which would be another BC break.